CommonMark/Formatters/HtmlFormatter.cs

```diff
@@ -217,6 +217,29 @@ protected virtual void WriteBlock(Block block, bool isOpening, bool isClosing, o
                     break;
 
                 case BlockTag.IndentedCode:
+
+                    if (isOpening)
+                    {
+                        EnsureNewLine();
+
+                        Write("<pre><code");
+                        if (Settings.TrackSourcePosition)
+                            WritePositionAttribute(block);
+
+                        Write('>');
+                    }
+
+                    if (0 == (_settings.AdditionalFeatures & CommonMarkAdditionalFeatures.EmphasisInIndentedCode))
+                    {
+                        ignoreChildNodes = true;
+                        WriteEncodedHtml(block.StringContent);
```

Comment:
it feels like this part should go inside (isOpening) branch - just to avoid the potential duplication of output in case the document tree is modified to have children in the indented code in some other case